homeassistant/components/systemmonitor/coordinator.py

```diff
@@ -209,6 +214,42 @@ def update_data(self) -> dict[str, Any]:
             _LOGGER.debug("processes: %s", processes)
             processes = list(processes)
```

Comment:
I guess we could also iterate through the process list and only copy those we are interested in (potentially exit the loop early once we have all processes monitored :thinking: ).